app/dashboard/views.py

```diff
@@ -1358,6 +1358,10 @@ def apitos(request):
     return TemplateResponse(request, 'legal/privacy.html', {})
 
 
+def labs(request):
+    return TemplateResponse(request, 'legal/privacy.html', {})
```

Comment:
legal/privacy.html?

Reply:
Ahh.. that was just temporary, I'm creating the template for it.